pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go

```diff
@@ -153,7 +153,7 @@ func (b *beResourceCollector) getBECPURealMilliLimit() (int, error) {
 
 func (b *beResourceCollector) getBECPURequestSum() resource.Quantity {
 	requestSum := int64(0)
-	for _, podMeta := range b.statesInformer.GetAllPods() {
+	for _, podMeta := range b.statesInformer.(*statesinformer.StatesInformerImpl).GetAllPods() {
```

Comment:
we can use the `PodsInformerPlugin` here, and beResourceCollector can be inited by statesInformer.GetInfromerPlugin()